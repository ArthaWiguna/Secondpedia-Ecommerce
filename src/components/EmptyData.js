import React from "react";
import emptyIcon from "../assets/img/empty-icon.svg";

const EmptyData = ({ message, imageWidth }) => {
  console.log(imageWidth);
  return (
    <div className="px-10 py-4">
      <img
        src={emptyIcon}
        alt="empty-icon"
        className={`block mx-auto sm:w-[${imageWidth}]`}
      />
      <p className="text-sm font-normal text-[#151515] text-center mt-6">
        {message}, <br /> sabar ya rejeki nggak kemana kok
      </p>
    </div>
  );
};

export default EmptyData;
