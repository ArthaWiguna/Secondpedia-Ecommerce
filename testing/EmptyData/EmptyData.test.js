import React from "react";
import renderer from "react-test-renderer";
import EmptyData from "../../src/components/EmptyData.js";

jest.mock("../../src/assets/img/empty-icon.svg", () => {
  return {
    default: "empty-icon.svg",
  };
});

it("renders correctly", () => {
  const tree = renderer.create(<EmptyData />).toJSON();
  expect(tree).toMatchSnapshot();
});
