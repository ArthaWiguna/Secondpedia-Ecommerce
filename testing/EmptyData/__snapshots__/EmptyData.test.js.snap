// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="px-10 py-4"
>
  <img
    alt="empty-icon"
    className="block mx-auto sm:w-[undefined]"
    src={
      Object {
        "default": "empty-icon.svg",
      }
    }
  />
  <p
    className="text-sm font-normal text-[#151515] text-center mt-6"
  >
    , 
    <br />
     sabar ya rejeki nggak kemana kok
  </p>
</div>
`;
